.github/workflows/dart-tests.yaml

```diff
@@ -258,7 +259,7 @@ jobs:
           ./wait-for-it.sh localhost:9091 -t 60 -- echo "### Redis is UP at :9091"
         working-directory: tests/docker/tests_integration
       - name: Run integration tests
-        run: dart test -t integration --reporter=failures-only
+        run: dart test -t integration -j=1 --reporter=failures-only
```

Comment:
FYI @SandPod

For now probably fine as each module only has 1-3 test files anyway.
If that changes, we might need to adapt the pattern from the `test_server`, to run only the needed test sequentially.